google-cloud-spanner/src/test/java/com/google/cloud/spanner/DatabaseAdminClientImplTest.java

```diff
@@ -473,17 +473,6 @@ public void createBackupWithBackupObject() throws ExecutionException, Interrupte
     assertThat(op.get().getId().getName()).isEqualTo(BK_NAME);
   }
 
-  @Test(expected = IllegalArgumentException.class)
```

Comment:
Instead of removing the test, should the test just be modified to indicate that `createBackup` can now be called without any expire time?